core/rfb.js

```diff
@@ -323,6 +318,12 @@ export default class RFB extends EventTargetMixin {
         }
     }
 
+    get showDot() { return this._showDot; }
```

Comment:
I would prefer if "cursor" is in here somewhere to make it clear what it refers to. E.g. `showCursorDot`.

Reply:
OK